src/CabanaPD_Input.hpp

```diff
@@ -62,10 +62,6 @@ class Inputs
             inputs["bulk_modulus"]["value"] = K;
```

Comment:
We may need a new PR for this, but the following does not seem correct. 

// Set approximate bulk modulus if not set by user
        // Used in timestep estimation
        if ( !inputs.contains( "bulk_modulus" ) )
        {
            if ( !inputs.contains( "elastic_modulus" ) )
                throw std::runtime_error( "Must input either bulk_modulus or "
                                          "elastic_modulus." );
            double E = inputs["elastic_modulus"]["value"];
            double nu = 0.25;
            double K = E / ( 3 * ( 1 - 2 * nu ) );
            inputs["bulk_modulus"]["value"] = K;
        }

The reason is that it will compute bulk modulus K assuming nu = 0.25, which is only valid for bond-based PD (PMB). Normally, one would input two constants (except for bond-based which would be just one) from the following: elastic modulus, bulk modulus, Poisson's ratio, and shear modulus.

Reply:
Fixed as you suggested